### PR TITLE
[IMP] website: redesign `s_picture` snippet

### PR DESCRIPTION
**Pull request by stefanorigano**
This PR/task is handled by @chgo-odoo 

---

The goal of this commit is to improve the visual design of the default `s_picture` snippet.

task-4047748

- Requires : https://github.com/odoo/design-themes/pull/824

| Master | This PR (with the new font pair) |
|--------|--------|
| image | image | 

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/views/snippets/s_picture.xml [hunk 1]
```diff
@@ -2,16 +2,15 @@
...
+    <section class="s_picture pt64 pb64">
         <div class="container">
-            <h2 class="display-3-fs" style="text-align: center;">A punchy Headline</h2>
-            <p style="text-align: center;">Choose a vibrant image and write an inspiring paragraph about it.<br/> It does not have to be long, but it should reinforce your image.</p>
-            <p style="text-align: center;"><br/></p>
+            <h2 style="text-align: center;">Step Up Your Game</h2>
+            <p style="text-align: center;">Experience unparalleled comfort, cutting-edge design, and performance-enhancing<br/>technology with this latest innovation, crafted to elevate every athlete's journey.</p>
             <div class="row">
-                <div class="col-lg-12 pb24" style="text-align: center;">
+                <div class="col-lg-12" style="text-align: center;">
```
Comment (stefanorigano) [inline addons/website/views/snippets/s_picture.xml:10]:
```suggestion
                <div class="col-lg-12 pt24" style="text-align: center;">
```

File: addons/website/views/snippets/s_picture.xml [hunk 2]
```diff
@@ -2,16 +2,15 @@
 <odoo>
 
 <template id="s_picture" name="Title - Image">
-    <section class="s_picture pt48 pb24 o_cc o_cc2">
+    <section class="s_picture pt64 pb64">
         <div class="container">
```
Comment (stefanorigano) [inline addons/website/views/snippets/s_picture.xml:6]:
```suggestion
        <div class="o_container_small">
```

**Review by stefanorigano — COMMENTED** [review]
Thanks @chgo-odoo , two minor design remarks, LGTM the rest.
Feel free to move to tech. review when done 👍 

Thanks!

**Comment by detrouxdev** [comment]
@chgo-odoo Could you quickly rebase this one to get a green runbot please? 🙏🏻  I'll merge after

**Comment by chgo-odoo** [comment]
Hello @detrouxdev, thanks for having a look at it !

I just rebased the branch, let me know if there is anything else missing on this one,

Thanks !

**Comment by detrouxdev** [comment]
@robodoo r+

**Pull request closed without merging**